includes/classes/Blocks.php

```diff
@@ -56,10 +56,11 @@ public function render_block( array $attributes ): string {
 		$response = $jobber->get_form( $form_type );
 
 		if ( is_wp_error( $response ) ) {
-			return sprintf(
-				'<p class="jobber-error">%s</p>',
-				esc_html( $response->get_error_message() )
-			);
+			// if we encounter an error when rendering on the front-end,
```

Comment:
I think we need to add the same thing on line 81 below, if we end up in a situation where no iframe URL is returned